### ISSUE TITLE
Notepad++ resets my auto-backup setting after I perform a git commit with it.

### ISSUE THREAD

**Issue by RomanTraistari** [notepad-plus-plus/notepad-plus-plus#5151]
### Description of the Issue
Notepad++ resets my auto-backup setting after I perform a git commit with it.
Have only been able to replicate this issue on the 32 bit version.

### Steps to Reproduce the Issue

1.  Have Notepad++ set as the default git editor:  
core.editor='C:\Program Files\Notepad++\notepad++.exe' -multiInst -nosession -noPlugin  
2. Have a session of Notepad++ open with multiple files open, including new unsaved files.
Auto-backup is enabled, every 10 seconds.
3. Perform a git commit, close commit message window.
4. Close also the other window with the open files.
5. Open Notepad++ again, check auto-backup settings.

### Expected Behavior

Auto-backup stays enabled, every 10 seconds.

### Actual Behavior
Auto-backup setting is disabled.

### Debug Information
Version 7.5
Can't provide more debug info at the moment as this only happens on the computer at work.

**Comment by Phroneris** [comment]
To launch new session causes resetting auto-backup setting. It's reported: https://github.com/notepad-plus-plus/notepad-plus-plus/issues/5063
Perhaps you would better use another text editor for Git till this bug is fixed.

**Comment by stuhlmeier** [comment]
I use 64-bit Notepad++ (7.7.1) and have the same issue.

**Comment by sasumner** [comment]
I believe I've read somewhere that having a portable copy of Notepad++ doing the "git editing" is a workaround to this.  So you'd have your "main" (installed) Notepad++ for most uses, then a portable copy that you point the git thing to for that specific editing use.

**Comment by donho** [comment]
> So you'd have your "main" (installed) Notepad++ for most uses, then a portable copy that you point the git thing to for that specific editing use.

Humm... The better solution would be "forget" this setting while `-multiInst` is used.

**Comment by sasumner** [comment]
@donho I think you should do it as I had a look and don't see what to do.

**Comment by caioproiete** [comment]
Thanks @sasumner for the workaround. I can confirm it works.

Steps:
- [Download](https://notepad-plus-plus.org/download/) the zip package or 7z package
- Extract it to a folder you want to keep (e.g. I used `C:\Users\caioproiete\tools\npp-for-git`)
- Open Git Bash and run:
```bash
git config --global core.editor "\"C:\\Users\\caioproiete\\tools\\npp-for-git\\notepad++.exe\" -multiInst -notabbar -nosession -noPlugin"
```

**Comment by sasumner** [comment]
FWIW, if I try the original "steps to reproduce" with Notepad++ 7.7.1 x64, I obtain the "Expected behavior" and not the "Actual behavior", so I'm not sure what to think about this....

**Comment by donho** [comment]
I cannot reproduce it anymore in v7.9.1.
Does anyone confirm it's fixed in v7.9.1 please ?

**Comment by ArkadiuszMichalski** [comment]
Maybe fixed by b3a6638?

**Comment by donho** [comment]
Fixed by https://github.com/notepad-plus-plus/notepad-plus-plus/commit/b3a6638b936c097c5323814fe046b6ddc75695cc
